src/lib/y2network/widgets/vlan_id.rb

```diff
@@ -23,7 +23,7 @@ def init
       end
 
       def store
-        @config["VLAN_ID"] = value
+        @config["VLAN_ID"] = value.to_s
```

Comment:
Could `value` be nil?

Reply:
no, it is input selection. Only possibility is if widget is disabled, but in that case store method is not called.